### ISSUE TITLE
Player model legs should turn with the player's movement direction

### ISSUE THREAD

**Issue by Paril** [jdolan/quetoo#390]
We're using Quake 3-style player models so we should definitely implement the legs turning in movement directions. It looks kinda weird to be strafing and having your legs pointing forward.

**Comment by Paril** [comment]
Ideally I'd like to have it look more like Quake 3, I loved the player animation in that (like the legs not turning unless they have to, torso/head doing the visual aiming).

**Comment by jdolan** [comment]
I *think* this might be a bit ambitious to get to for 1.0, but I'll leave that up to you :)

**Comment by Paril** [comment]
I didn't think it was ambitious until I looked at how you implemented tags. I cried a bit :(
I did get a semi-working version of this going (legs didn't pitch, the head and body pitched, and it was all properly offset by the tag origin), but it wasn't stable enough for me to push (looking down slightly in third person caused the whole upper body to disappear). Having the legs work exactly like they do in Q3 will be hard because it requires some persistent state between the Cg player entities, and we don't have a good spot for that yet (maybe it can be thrown in client_info?).
